components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OIDCAuthenticatorConstants.java

```diff
@@ -1,17 +1,17 @@
-/**
- * Copyright (c) 2015, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+/*
+ * Copyright (c) 2015, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Changed year range should come in

Reply:
Addressed with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/184